trunk/mailer.http.class.php

```diff
@@ -75,10 +75,7 @@ protected function get_request_body()
                 break;
         }
 
-        $replyTos = $this->getReplyToAddresses();
-        if (count($replyTos)) {
-            $body['content']['reply_to'] =  $replyTos[0];
```

Comment:
That's definitely worthy. If we've multiple values for same header, we can do that. Do you want to handle this in this PR?